config/storage.yml

```diff
@@ -6,10 +7,13 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
+# yamllint disable-line rule:line-length
```

Comment:
do we really need to disable for comments?

Reply:
No, we don't